### PR TITLE
fix(DB/creature_template):  Fix Twilight Fissure interaction

### PR DESCRIPTION
**Pull request by Si1ker**
This change will prevent the chance of the Twilight Fissure to not go in accordance with the visual=damage

## CHANGES PROPOSED:
-  Add unit_flag 33554432
-  Add ScriptName "trigger_periodic" as other zone effects triggers

## ISSUES ADDRESSED:
- There are some times where the fissure can follow for a little bit a layer so the damage could not go in accordance with the actual effect.
image

## TESTS PERFORMED:
- Tested in-game

## HOW TO TEST THE CHANGES:

Apply SQL on db
Go fight with Vesperon as Sartharion is currently broken on last rev, see #3633 
No matter how many encounters yo do the fissure is static and the damage goes in accordance with the visual zone

## KNOWN ISSUES AND TODO LIST:

- [ ]
- [ ] 

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by Kitzunu** [comment]
Does the Rooted inhabittype work? 
Last I checked it was not working

**Comment by Kitzunu** [comment]
Yea not implemented x)

2020-10-31 22:45:24 Creature (Entry: 30641) has wrong value (8) in `InhabitType`, creature will not correctly walk/swim/fly.

**Comment by Si1ker** [comment]
@Kitzunu  Thanks, gonna check.

**Comment by Si1ker** [comment]
@Kitzunu  ready

**Comment by Kitzunu** [comment]
When you too lazy to implement the proper flag x) I'd do the same xddd

**Comment by Si1ker** [comment]
@Kitzunu I've updated the PR description so everything is clear, if u wanna approve I won't get mad 🐱 
Tested second time, working !

**Comment by Kitzunu** [comment]
All I want to do is sleep.
Which imma do now x)

**Review by Helias — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1604182926047452600.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1604182926047452600');
+
+/*
+ * Update by Silker | <www.azerothcore.org> | Copyright (C)
+*/
+
+UPDATE `creature_template` SET `unit_flags`=`unit_flags`|33554432 WHERE  `entry` IN
+(30641, 31521);
+UPDATE `creature_template` SET `ScriptName` = "trigger_periodic" WHERE `entry`= 30641;
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1604182926047452600.sql:9]: it should be `'trigger_periodic'`, not `"trigger_periodic"`

Comment (Si1ker) [reply data/sql/updates/pending_db_world/rev_1604182926047452600.sql]: @FrancescoBorzi done 🐱

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by iyasenai** [comment]
I have tested the changes, it works correctly :)

https://www.youtube.com/watch?v=hmRST94qCk8

**Comment by Si1ker** [comment]
Thanks once again @iyasenai  !! 😸

**Pull request merged**